README.md

```diff
@@ -38,6 +38,9 @@ GOOGLE_APP_CLIENT_EMAIL=your-client-email
 * `npm start` for development mode, or
 * `npm run build` then `npm run prod-server` for production mode
 
+### Installing on Apple Silicon
+Since `canvas` does not provide pre-built binaries for Apple chips during `npm install` it will try to compile it manually. To get this to succeed you new to install several dependencies by following the instructions [here](https://github.com/Automattic/node-canvas#compiling). 
```

Comment:
```suggestion
Since `canvas` does not provide pre-built binaries for Apple chips during `npm install` it will try to compile it manually. To get this to succeed you need to install several dependencies by following the instructions [here](https://github.com/Automattic/node-canvas#compiling). 
```